app/views/overrides/activation_keys/_host_environment_select.html.erb

```diff
@@ -1,5 +1,7 @@
 <% f ||= form %>
-<%= javascript "katello/hosts/host_and_hostgroup_edit" %>
+<% content_for(:javascripts) do -%>
+  <%= javascript "katello/hosts/host_and_hostgroup_edit" %>
+<% end %>
```

Comment:
this is redundant, `<%= javacript …` already does `content_for(:javascripts)`: https://github.com/theforeman/foreman/blob/262b5caceab9c3f71d0b80f189cc724389593fb0/app/helpers/layout_helper.rb#L98-L100